src/components/logo/logo.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Heading,
+  VStack,
+  HStack,
+  Logo,
+  Link,
+} from "@nypl/design-system-react-components";
+import React from "react";
+import appConfig from "appConfig";
+
+const DCLogo = () => {
+  return (
+    <Link
+      isUnderlined={false}
+      aria-label={"Digital Collections Homepage"}
+      href={`${appConfig.DC_URL}`}
+    >
+      <HStack>
+        <Logo
+          pt="2"
+          pb="2"
+          name="nyplLionBlack"
+          size="xxsmall"
+          aria-label="NYPL Lion Logo"
+          sx={{
+            display: { sm: "block", md: "none", lg: "none" },
+          }}
+        />
+        <Logo
+          name="digitalCollectionsBlack"
+          aria-label="NYPL Digital Collections Logo"
+          sx={{
+            display: { sm: "none", md: "none", lg: "block" },
+          }}
+        />
+        {/* <VStack
```

Comment:
Is there a reason to keep this comment block?

Reply:
See above discussion! Figuring out how to override DS logo size